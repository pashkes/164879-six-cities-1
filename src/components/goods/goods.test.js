import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Goods from "./goods.jsx";

configure({adapter: new Adapter()});

it(`goods is renders correctly`, () => {
  const goods = shallow(<Goods list={[``]} />);
  expect(goods).toMatchSnapshot();
});
