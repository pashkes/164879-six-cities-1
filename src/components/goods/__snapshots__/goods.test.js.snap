// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`goods is renders correctly 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
    key="0"
  />
</ul>
`;
